browser/src/slideshow/PauseTimer.ts

```diff
@@ -122,80 +118,11 @@ class PauseTimerGl extends Transition2d implements PauseTimer {
 		}
 	}
 
-	// TODO: We can replace with loadTexture from RenderContext
-	private load2dCanvasToGlCanvas(canvas: HTMLCanvasElement): WebGLTexture {
-		const texture = this.gl.createTexture();
-		if (!texture) {
-			throw new Error('Failed to create texture');
-		}
-		this.gl.bindTexture(this.gl.TEXTURE_2D, texture);
-		this.gl.texImage2D(
-			this.gl.TEXTURE_2D,
-			0,
-			this.gl.RGBA,
-			this.gl.RGBA,
-			this.gl.UNSIGNED_BYTE,
-			canvas,
-		);
-		this.gl.texParameteri(
-			this.gl.TEXTURE_2D,
-			this.gl.TEXTURE_WRAP_S,
-			this.gl.CLAMP_TO_EDGE,
-		);
-		this.gl.texParameteri(
-			this.gl.TEXTURE_2D,
-			this.gl.TEXTURE_WRAP_T,
-			this.gl.CLAMP_TO_EDGE,
-		);
-		this.gl.texParameteri(
-			this.gl.TEXTURE_2D,
-			this.gl.TEXTURE_MIN_FILTER,
-			this.gl.LINEAR,
-		);
-		this.gl.texParameteri(
-			this.gl.TEXTURE_2D,
-			this.gl.TEXTURE_MAG_FILTER,
-			this.gl.LINEAR,
-		);
-
-		return texture;
-	}
-
-	public render(): void {
-		this.gl.viewport(
-			0,
-			0,
-			this.context.canvas.width,
-			this.context.canvas.height,
-		);
-		this.gl.clearColor(0.0, 0.0, 0.0, 1.0);
-		this.gl.clear(this.gl.COLOR_BUFFER_BIT);
-
-		this.gl.useProgram(this.program);
-
-		this.gl.activeTexture(this.gl.TEXTURE0);
-		this.gl.bindTexture(this.gl.TEXTURE_2D, this.textTexture);
-
-		const textureLocation = this.gl.getUniformLocation(
-			this.program,
-			'u_texture',
+	private getPauseTextContent(): string {
+		return _('Pause... ( %SECONDS% )').replace(
```

Comment:
@timar is that correct format for parameter inside translatable string? I remeber there was some change recently...